devstack/lib/nfp

```diff
@@ -238,3 +203,113 @@ function copy_nfp_files_and_start_process {
     echo "Configuration success ... "
 
 }
+
+function configure_nfp_loadbalancer {
+    echo "Configuring NFP Loadbalancer plugin driver"
```

Comment:
Keep this method in plugin.sh as all the config related things are there in that file.
And use iniset kind of configuration instead of sed